bugfix/disable-search-service (#7)
fivetran/dbt_unified_rag: README.md

## What does this dbt package do?

<!--section="unified_rag_transformation_model"-->

> The main focus of this dbt package is to generate an end model and which contains the below relevant unstructured document data to be used for Retrieval Augmented Generation (RAG) applications leveraging Large Language Models (LLMs):

Reviewer: ```suggestion
The main focus of this dbt package is to generate an end model that contains the below relevant unstructured document data to be used for Retrieval Augmented Generation (RAG) applications leveraging Large Language Models (LLMs):
```
